### ISSUE TITLE
Stuck forever in  "Compiling..." step after "sanity start" command

### ISSUE THREAD

**Issue by giemaz** [sanity-io/sanity#3255]
Title says all. I yesterday i created a new sanity project without editing single code line. I run it for test and everything was ok. But today it stuck right after star in terminal at "compiling... " step. Same with new projects I created after.

**Comment by wlululululua** [comment]
Me too. Did you solve this problem?
My node version is v16.13.0 and my npm version is v8.3.0. I never successed run "sanity start".

**Comment by giemaz** [comment]
> Me too. Did you solve this problem? My node version is v16.13.0 and my npm version is v8.3.0. I never successed run "sanity start".

Yes. My rookie mistake was that I created a project not in the same hard drive where is node and operating system installed. If this is your case too, just copy the project there and try run again.

**Comment by wlululululua** [comment]
> > Me too. Did you solve this problem? My node version is v16.13.0 and my npm version is v8.3.0. I never successed run "sanity start".
> 
> Yes. My rookie mistake was that I created a project not in the same hard drive where is node and operating system installed. If this is your case too, just copy the project there and try run again.

Thank you very much!!! I'm windows user, and I test in drive C, it doesn't work, and then I test in drive D where node installed in, it successed finally. You saved me, thanks again!!!

**Comment by kitovvska** [comment]
My node is installed in drive C like my project is created also. Still stuck forever in "Compiling..." :( anyone can help me?

**Comment by danxqhu** [comment]
> > Me too. Did you solve this problem? My node version is v16.13.0 and my npm version is v8.3.0. I never successed run "sanity start".
> 
> Yes. My rookie mistake was that I created a project not in the same hard drive where is node and operating system installed. If this is your case too, just copy the project there and try run again.

This works! Thank you!

**Comment by rexxars** [comment]
This should now be improved as of [Sanity Studio v3](https://www.sanity.io/blog/sanity-studio-v3-simplified-yet-powerful-customization).
